inc/class-command.php

```diff
@@ -195,7 +195,15 @@ protected function run_command( $command ) {
 	 * Command to start the virtual machine
 	 */
 	protected function start( InputInterface $input, OutputInterface $output ) {
-		return $this->run_command( 'vagrant up' );
+		$hosts = $this->get_project_hosts();
```

Comment:
I think this method might be from an older version of the package 😬 sorry about that! I'll remake it to get this working.